.circleci/config.yml

```diff
@@ -553,7 +553,7 @@ jobs:
 
   test-frontend-2:
     docker: &docker-node-browsers-image
```

Comment:
if this works we can merge &docker-node-image and &docker-node-browsers-image after